stdlib/2/typing.pyi

```diff
@@ -385,7 +385,8 @@ class Pattern(Generic[AnyStr]):
 
 # Functions
 
-def get_type_hints(obj: Callable) -> dict[str, Any]: ...
+def get_type_hints(obj: Callable, globalns: Optional[dict[AnyStr, Any]],
```

Comment:
To be clear, using AnyStr means that all the keys must be either bytes or Text; you can't have a mix.